contents/tutorials/how-to-connect-discord-to-posthog-with-zapier.md

```diff
@@ -6,7 +6,7 @@ featuredImage: ../images/tutorials/banners/discord-tutorial-banner.png
 featuredTutorial: false
 date: 2022-05-04
 author: ['joe-martin']
-topics: ['plugins', 'zapier']
+topics: ['apps']
```

Comment:
Huh. If they're getting pulled in here then we should definitely remove them. 

@corywatilo Maybe I've misunderstood something here. Is there a way to tag a tutorial such that it can be pulled into the relevant `/app` page, but doesn't get added to this sidebar?

Reply:
Might need to use a different tag in front matter if you want to do that. The "topics" were designed specifically for tutorial categories, so anything listed in topics will get aggregated to this list as you're seeing here.